### PR TITLE
editorconfig file to not use the one from stratagus

### PR DESCRIPTION
**Pull request by zzam**
editorconfig searches parent directories when not configured to stop

See https://EditorConfig.org

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# top-most EditorConfig file
+root = true
```
Comment (Jarod42) [inline .editorconfig:2]: Comment to specify that it doesn't specify any editor config might be fine IMO.
(Is it right? There are no default config in that case?)

Comment (zzam) [reply .editorconfig:2]: Done.
I hope this is fine.

**Comment by zzam** [comment]
In case some folders are subject to manual changing (guichan?) an separate .editorconfig file can be added there.

File: .editorconfig (hunk 1 above)
Comment (Jarod42) [reply .editorconfig:2]: I meant, "Formatting disabled"

**Pull request merged**